Sources/MapboxMaps/Viewport/States/FollowPuck/FollowPuckViewportStateOptions.swift

```diff
@@ -23,18 +23,6 @@
     /// not be modified.
     public var pitch: CGFloat?
 
-    /// The duration of an animation that happens once when
-    /// ``FollowPuckViewportState/startUpdatingCamera()`` is invoked.
-    ///
-    /// - Note: This option and the animation that it influences may be removed in a future update after
-    ///         a solution to the "moving target problem" is implemented in
-    ///         ``DefaultViewportTransition``. At the moment,
-    ///         ``DefaultViewportTransition`` calculates its animations based on the puck
-    ///         location at the *beginning* of the transition, so the farther the puck moves while the
-    ///         transition is in progress, the larger the jump when it completes and control is transferred
-    ///         to the target state. Tune this value for your use case to reduce the visibility of that jump.
-    public var animationDuration: TimeInterval
```

Comment:
shall we deprecate and then remove?

Reply:
Happy to change it. When would we remove? Next minor?

Reply:
yes, that would be good, otherwise, the v10.5.0-beta.1 will be broken, if we don't introduce the fix before the release date.

Reply:
Ah, makes sense.